libs/ui-components/src/components/Device/DevicesPage/EnrollmentRequestList.tsx

```diff
@@ -46,7 +46,7 @@ type EnrollmentRequestListProps = { refetchDevices?: VoidFunction; isStandalone?
 
 const EnrollmentRequestList = ({ refetchDevices, isStandalone }: EnrollmentRequestListProps) => {
   const { t } = useTranslation();
-  const [canApprove] = useAccessReview(RESOURCE.ENROLLMENT_REQUEST_APPROVAL, VERB.POST);
+  const [canApprove] = useAccessReview(RESOURCE.ENROLLMENT_REQUEST_APPROVAL, VERB.PUT);
```

Comment:
the RBAC verb did not change in the backend yet. It is still `post`

Reply:
You are right, I'm reverting the changes to RBAC.